yaksh/templates/yaksh/course_forum.html

```diff
@@ -0,0 +1,115 @@
+                <button type="button" class="close" data-dismiss="modal">&times;</button>
+              </div>
+              <div class="modal-body">
+                <form action="." method="POST" enctype='multipart/form-data'>
+                    <div class="form-group">
+                        {% csrf_token %}
+                        {{form}}
+                    </div>
+                    <input type="submit" class="btn btn-primary" value="Create Post">
+                </form>
+              </div>
+              <div class="modal-footer">
+                <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
+              </div>
+            </div>
+            
+          </div>
+        </div>
+        <br>
+        <br>
+        {% if posts %}
+            <table id="posts_table" class="tablesorter table">
+                <thead class="thread-inverse">
+                    <tr>
+                        <th width="700">Questions</th>
+                        <th>Created by</th>
+                        <th>Replies</th>
+                        <th>Last reply</th>
+                        <th></th>
+                    </tr>
+                </thead>
+                <tbody>
+                    {% for post in posts %}
+                        <tr>
+                            <td>
+                                <a href="{% url 'yaksh:post_comments' course.id post.uid %}">{{post.title}}</a>
+                                <small class="text-muted d-block">{{ post.description|truncatewords:30 }}</small>
+                                <small class="text-muted"><strong>Last updated: {{post.modified_at}}</strong></small>
+                            </td>
+                            <td>{{post.creator.username}}</td>
+                            <td>{{post.get_comments_count}}</td>
+                            <td>
+                                {% with post.get_last_comment as last_comment %}
+                                    {% if last_comment %}
+                                        {{last_comment.creator}}
+                                    {% else %}
+                                        None
+                                    {% endif %}
+                                {% endwith %}
+                            </td>
+                            <td>
+                                {% if user.profile.is_moderator %}
+                                    <small><a href="{% url 'yaksh:hide_post' course.id post.uid %}" class="pull-right btn btn-danger">Delete</i></a></small>
+                                {% endif %}
+                            </td>
+                        </tr>
+                    {% endfor %}
+                </tbody>
+            </table>
+        {% else %}
+            No discussion posts are there yet. Create one to start discussing.
+        {% endif %}
+        {% include "yaksh/paginator.html" %}
```

Comment:
Include paginator before posts list too because the user has to scroll down just for going to the next page.